src/packages/cli/src/import/index.ts

```diff
@@ -149,11 +150,29 @@ export const importDataSource = async (
 
 		spinner.stop();
 
+		let fileCount = 0;
 		for (const file of files) {
 			createDirectories(path.join('./src/', file.path));
-			writeFileSync(path.join(process.cwd(), './src/', file.path, file.name), file.contents);
+
+			const fileFullPath = path.join(process.cwd(), './src/', file.path, file.name);
+			let overwrite = true;
+			if (!overwriteAllFiles && file.needOverwriteWarning && existsSync(fileFullPath)) {
+				const prompt = await inquirer.prompt<{ overwrite: boolean }>([
+					{
+						type: 'confirm',
+						name: 'overwrite',
+						message: `Overwrite this file ${path.join(file.path, file.name)}?`,
+						default: true,
+					},
+				]);
+				overwrite = prompt.overwrite;
+			}
+			if (overwrite) {
+				writeFileSync(fileFullPath, file.contents);
+				fileCount += 1;
+			}
 		}
-		console.log(`${files.length} files have been successfully created in the project.`);
+		console.log(`${fileCount} files have been successfully created in the project.`);
```

Comment:
Minor thing and only if you think it makes sense, but can we say "created" if there's no files overwritten, and "overwritten" if there were files to overwrite?